conf/mesos-site.xml

```diff
@@ -89,6 +89,21 @@
     <value>3</value>
   </property>
 
+  <property>
+    <name>mesos.hdfs.jre-url</name>
+    <value>https://downloads.mesosphere.io/java/jre-7u76-linux-x64.tar.gz</value>
```

Comment:
@elingg: Ok.  I'll do that.  Any ideas for what would be an appropriate DEFAULT_JRE_URI or DEFAULT_LD_LIBRARY_PATH or DEFAULT_JRE_VERSION?  Just point at some url/path/version which I know doesn't exist?